inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -444,20 +444,9 @@ class Core::Impl : public ICore, public std::enable_shared_from_this<ICore> {
                 }
             }
 
-            // check extensions
-            auto extensionsNode = pluginNode.child("extensions");
-            std::vector<FileUtils::FilePath> listOfExtentions;
-
-            if (extensionsNode) {
-                FOREACH_CHILD(extensionNode, extensionsNode, "extension") {
-                    FileUtils::FilePath extensionLocation = FileUtils::toFilePath(GetStrAttr(extensionNode, "location").c_str());
-                    listOfExtentions.push_back(extensionLocation);
-                }
-            }
```

Comment:
should should not just remove this code, we need to put extensions to `this->extensions` for cases if someone used this functionality and put extensions to `plugin` xml tag